README.md

```diff
@@ -164,6 +169,10 @@ jobs:
 
 # Version history
 
+## v0.4
+
+  * Added support for specifying the machine architecture (x64 or ia32).
```

Comment:
Perhaps remove the explicit mention of `x64` and `ia32` here.

Reply:
Yes, one sec!